packages/csv-parser-price/src/main.js

```diff
@@ -183,9 +183,11 @@ export default class CsvParserPrice {
 // Easiest way to wrap the getCustomTypeDefinition in the memoize method
 CsvParserPrice.prototype.getCustomTypeDefinition = memoize(
   function _getCustomTypeDefinition (customTypeKey) {
-    const getTypeByKeyUri = createRequestBuilder().types
+    const getTypeByKeyUri = createRequestBuilder({
```

Comment:
Yeah, the fact that it uses it doesn't still validate the change

Do we change the module everytime another package is updated?

`csv-parser-price` uses `lodash` for example, sure we don't update our module everytime that gets updated

So you should see where I am driving to with this...